packages/wonder-blocks-data/src/util/__tests__/memory-cache.test.js

```diff
@@ -306,22 +377,70 @@ describe("MemoryCache", () => {
 
             // Act
             const result = cache.removeAll(fakeHandler);
-            const after = cache.cloneData();
 
             // Assert
             expect(result).toBe(3);
-            expect(after).toStrictEqual({
-                MY_HANDLER: {},
-                OTHER_HANDLER: {
+        });
+    });
+
+    describe("#cloneData", () => {
+        it("should return a copy of the cache data", () => {});
+
+        it("should throw if there is an error during cloning", () => {
+            // Arrange
+            const cache = new MemoryCache({
+                MY_HANDLER: {
                     MY_KEY: {data: "data!"},
                 },
             });
+            jest.spyOn(JSON, "stringify").mockImplementation(() => {
+                throw new Error("BANG!");
+            });
+
+            // Act
+            const act = () => cache.cloneData();
+
+            // Assert
+            expect(act).toThrowErrorMatchingInlineSnapshot(
+                `"An error occurred while trying to clone the cache: Error: BANG!"`,
+            );
         });
     });
 
-    describe("#cloneData", () => {
-        it("should return a copy of the cache data", () => {});
+    describe("@inUse", () => {
```

Comment:
Is the `#` for methods and `@` for getters a personal pattern or is this a "standard" of some sort?